chart/gvm/templates/nvt-plugins-sync-hook.yaml

```
@@ -20,7 +20,7 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ tpl .Values.image.openvas.registry . }}/{{ tpl .Values.image.openvas.repository . }}:{{ tpl .Values.image.openvas.tag . }}"
           imagePullPolicy: {{ .Values.image.openvas.pullPolicy }}
-          command: ["greenbone-nvt-sync"]
+          command: ["greenbone-nvt-sync --rsync"]
```

You cannot add `--rsync` as part of the string. The right choice is:

```suggestion
          command: ["greenbone-nvt-sync", "--rsync"]
```

or even better:

```suggestion
          command: ["greenbone-nvt-sync"]
          args: ["--rsync"]
```